### DISCUSSION TITLE
Gitstream Automatically Link to the Related Azure Boards Resource not working.

### DISCUSSION THREAD

**Discussion by thiagoalmeidariskadviser** [linear-b/gitstream#533] [Troubleshooting]
Recently I did the free integration of Gitstream to Github actions using the example [Automatically Link to the Related Azure Boards Resource](https://docs.gitstream.cm/integrations/azure-boards/#automatically-link-prs-to-related-azure-boards-resources).
However I am unable to show Issue Tracker Link of Azure Board.
Does anyone know what’s wrong with my gitstream.cm model?

Sample Task Azure Board
![image](https://github.com/linear-b/gitstream/assets/145681788/61802549-a456-4f19-b127-af693176c14a)

![image](https://github.com/linear-b/gitstream/assets/145681788/1492bc2f-dc9d-4b5e-b25e-14258a034beb)

[sample_automation_2.txt](https://github.com/user-attachments/files/15810895/sample_automation_2.txt)

**Comment by vim-zz** [comment]
Does your Azure board require some authentication to access it? or is it a public one?

@BenLloydPearson can you take a look?

**Reply by thiagoalmeidariskadviser** [reply]
Yes, require some authentication to access. 
It is currently working to relate the PR with AB#1234, but using the same config as I sent in the file on the Gitstream site does not work. I do not know if you need to inform any other configuration than what is to inform in the file.
